README.md

```diff
@@ -25,44 +27,6 @@ to activate the pre-commit check.
 
 Run the python script `main.py` located in the root of the repository.
 
-## [Wiki](https://github.com/ess-dmsc/nexus-constructor/wiki)
-
-Contains developer notes, general QML approaches
-
-## Running unit tests
-
-Unit tests are written to use [pytest](https://docs.pytest.org/en/latest/).
-Once the dependencies have been installed, they can be run from a terminal in
-the project's root directory by running the command `pytest`.
-
-Test coverage can be checked by running the following from the root of the repository:
-```
-pytest --cov=nexus_constructor
-```
-
-## Linter
-
-flake8 is used to check [pep8](https://www.python.org/dev/peps/pep-0008/?) 
-compliance. It is installed via pip and can be run as `flake8` in the project's 
-root directory. 
-
-## Testing the UI
-
-A script for testing the UI's functionality can be found at [tests/UITests.md](tests/UITests.md)
-
-## Building a distributable version
-
-A distributable version of the app, with the required python interpreter and
-libraries included can be built using [cx_Freeze](https://cx-freeze.readthedocs.io).
-It is included in the project's requirements file, and these must be installed
-in order to build the distributable. A build can be run with the following
-command in the projects root directory:
-```
-python setup.py build_exe
-```
-This will create the executable and copy it's required files to the `build`
-subdirectory
-
-cx_Freeze is capable of building distributable versions for Windows, OS X, and
-Linux, but can only do so from a machine running that operating system.
+## Developer Documentation
 
+See the [Wiki](https://github.com/ess-dmsc/nexus-constructor/wiki) for developer documentation.
```

Comment:
can you link this to https://github.com/ess-dmsc/nexus-constructor/wiki/Developer-Notes